projects/storefrontlib/src/layout/main/storefront.component.html

```diff
@@ -1,4 +1,4 @@
-<cx-asm *cxFeature="'asm'"></cx-asm>
+<cx-page-layout section="topHeader"></cx-page-layout>
```

Comment:
Can we just use this:
`<cx-page-slot position="TopHeaderSlot"></cx-page-slot>`. So, it does match this `<cx-page-slot position="BottomHeaderSlot"></cx-page-slot>`

Reply:
Fixed.